setup/coder-for-docker/upgrade.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Upgrade"
+description: Learn how to upgrade your Coder for Docker deployment.
+---
+
+This guide will show you how to upgrade your Coder for Docker deployment.
+
+To upgrade, run the following command to download the resources you need,
+including the latest images (ensure that you're providing the correct version
+number in the command):
+
+```console
+docker run --rm -it \
+    -p 7080:7080 \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -v ~/.coder:/var/run/coder \
+    codercom/coder:1.29.0
```

Comment:
I understand we have to update this as we walk forward.

I wonder if we want to make it like this to make it obvious it can be changed? Either way is fine. Just providing an alternative

```console
VERSION=1.29.0 docker run --rm -it \
    -p 7080:7080 \
    -v /var/run/docker.sock:/var/run/docker.sock \
    -v ~/.coder:/var/run/coder \
    codercom/coder:$VERSION
```